cluster/manager_delete.go

```diff
@@ -45,10 +46,10 @@ func (m *Manager) DeleteCluster(ctx context.Context, cluster CommonCluster, forc
 		return err
 	}
 	var timer *prometheus.Timer
-	if viper.GetBool("metrics.debug") {
-		timer = prometheus.NewTimer(StatusChangeDuration.WithLabelValues(cluster.GetCloud(), cluster.GetLocation(), pkgCluster.Deleting, org.Name, cluster.GetName()))
+	if viper.GetBool(config.MetricsDebug) {
```

Comment:
Factor out the creation of prometheus timer into a function of `Manager`. This function can be used from manager_delete.go as well

Reply:
thx, fixed